spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RetryingMessageListenerAdapter.java

```diff
@@ -37,7 +40,7 @@
  */
 public class RetryingMessageListenerAdapter<K, V>
```

Comment:
Yes, I believe there is something in the reference about it  - we don't know where in the batch to restart from; they are better off using a retry template within their listener.